paint.net-plugin-pyrochild: Add version 2017-12-04

NotWearingPants opened the pull request:
Added a very popular paint.net plugin pack by pyrochild, by copying the dll files into paint.net's `Effects` folder

Ash258 commented on bucket/paint.net-plugin-pack-pyrochild.json:
```diff
@@ -0,0 +1,19 @@
+{
+    "version": "2017-12-04",
+    "homepage": "https://forums.getpaint.net/topic/7291-pyrochild-plugins",
+    "description": "pyrochild Plugins for paint.net",
+    "depends": "paint.net",
+    "url": "https://forums.getpaint.net/applications/core/interface/file/attachment.php?id=13067#/dl.zip",
+    "hash": "dc05dd0604d64481ce44d35156a27ec2c6fb53b4a902aeb89cfe521cb0ef699c",
+    "installer": {
+        "script": [
+            "(Get-ChildItem \"$dir\\*.dll\").Name | Out-File \"$dir\\plugins.txt\"",
+            "Move-Item \"$dir\\*.dll\" \"$(appdir paint.net)\\current\\Effects\\\""
+        ]
+    },
+    "uninstaller": {
+        "script": [
+            "Get-Content \"$dir\\plugins.txt\" | ForEach-Object { Remove-Item \"$(appdir paint.net)\\current\\Effects\\$_\" }"
+        ]
+    }
+}
```
```suggestion
{
    "version": "2017-12-04",
    "description": "Collection of paint.net effects by pyrochild",
    "homepage": "https://forums.getpaint.net/topic/7291-pyrochild-plugins",
    "license": "MIT",
    "depends": "paint.net",
    "url": "https://forums.getpaint.net/applications/core/interface/file/attachment.php?id=13067#/dl.zip",
    "hash": "dc05dd0604d64481ce44d35156a27ec2c6fb53b4a902aeb89cfe521cb0ef699c",
    "pre_install": [
        "$ef = Join-Path (appdir 'paint.net' $global) 'current\\Effects'",
        "(Get-ChildItem \"$dir\\*.dll\").Name | Out-File \"$dir\\plugins.txt\" -Encoding Ascii",
        "Copy-Item \"$dir\\*.dll\" $ef -Force"
    ],
    "uninstaller": {
        "script": "Get-Content \"$dir\\plugins.txt\" | Remove-Item -Path { Join-Path (appdir 'paint.net' $global) \"current\\Effects\\$_\" } -ErrorAction SilentlyContinue"
    }
}

```

NotWearingPants commented:
> * Always open 1 PR per manifest!

Didn't know that was a rule, consider adding a `CONTRIBUTING.md` file mentioning that

> * https://github.com/lukesampson/scoop-extras/blob/master/.editorconfig

The manifests I added had CRLF line endings, 4 space indents, and had a newline at the end.
What exactly did I miss? Please be more thoughtful, I understand you have a lot of work, but I'm only trying to help.

> * `git mv bucket/paint.net-plugin-pack-pyrochild.json  bucket/paint.net-pyrochild-plugin.json`

I disagree, using the `paint.net-plugin` prefix it will be much easier to find all the available plugins.
All of the `keepass` plugins are also formatted as `keepass-plugin-xxx.json`.
Also, this is not a single plugin, this is a pack of multiple plugins. Therefore I added the `pack` prefix as well.
Thoughts?

Ash258 commented:
Did you ever saw update log which contains something like that name you provided as a PR title with multiple manifests??

> The manifests I added had CRLF line endings, 4 space indents, and had a newline at the end. What exactly did I miss? Please be more thoughtful, I understand you have a lot of work, but I'm only trying to help.

https://github.com/lukesampson/scoop-extras/pull/4264/commits/d4d7708fd96305e3441113da97bbbe15ef5c2c1a 

`git mv bucket/paint.net-plugin-pack-pyrochild.json bucket/paint.net-plugin-pyrochild.json`

NotWearingPants commented:
Done :)

Ash258 commented:
/verify
